python_ta/cfg/visitor.py

```diff
@@ -12,14 +12,16 @@ class CFGVisitor:
         The top of the stack corresponds to the end of the list.
         (compound statement [while], {'Break'/'Continue': CFGBlock to link to})
     """
-    cfg: ControlFlowGraph
+    cfgs: List[ControlFlowGraph]
```

Comment:
This is a really good start. However, instead of a list, I'd like you to use a dictionary `Dict[Node, ControlFlowGraph]`, where the key is a `Module` or `FunctionDef` node, and the corresponding value is the CFG for that node. This will be helpful in identifying these different graphs later.